### ISSUE TITLE
Cycle at least 4380 km in 2023

### ISSUE THREAD

**Issue by Daniel-Mietchen** [Daniel-Mietchen/ideas#1746]
As per Resolution 2.vii of https://github.com/Daniel-Mietchen/ideas/blob/master/new-year-resolutions/2023.md : 

> Make more room for physical activity, e.g. commit to cycling at least as many km as there are hours with daylight over a certain period of time.

**Comment by Daniel-Mietchen** [comment]
By now, I have decided that I will go for cycling, and that the "certain period" will be the entire year. At a yearly average of 12h of daylight per day, this gives 12(h/d)*365d*1(km/h)= 4380 km.

So far, I have cycled about 10km per week this year. I am thus planning to ramp this up to an average of about one order of magnitude higher and then try to sustain it over the course of the year, especially during times when there are more daylight hours per day.

I have not yet found a good way to track these kilometers at a yearly scale - the simple option of an odometer is vulnerable to theft, and I am not aware of good apps for this at the moment.
